app/config.py

```diff
@@ -0,0 +1,35 @@
+import os
+import testing.postgresql
+import psycopg2
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config:
+    SECRET_KEY = os.getenv('SECRET_KEY', 'Fhhty%$#*hghghf$Tf')
+    DEBUG = False
+
+
+class DevelopmentConfig(Config):
+    DEBUG = True
+    DATABASE_URL = 'postgres://postgres@localhost:5432/postgres/'
+
+
+#class TestingConfig(Config):
+ #   Postgresql = testing.postgresql.Postgresql()
+  #  DEBUG = True
+   # TESTING = True,
+    #print('>>>>>>>>>>>>>>>>>>', Postgresql.url())
+    #DATABASE_URL = Postgresql.url()
+
```

Comment:
Why is this commented out?

Reply:
i had commented it out to test on something but i have rectified that its no longer commeted